examples/custom_container.py

```diff
@@ -3,6 +3,9 @@
 from pyAitu.models import Message, Options, Form, Header, FormClosed, \
     FormSubmitted, FormAction, CustomContainer, Indent, \
     FlexOptions, Image, FileMetadata, Text, Divider
+from pyAitu.models.form.alignment import RIGHT
```

Comment:
```suggestion
from pyAitu.constants import RIGHT
```
I think its nice to have import from one folder, model.form.alignment may cause confusions

Reply:
moved all constants to package models.constants